lib/pangea/widgets/chat/overlay_header.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:fluffychat/pages/chat/chat.dart';
+import 'package:fluffychat/pages/chat/chat_app_bar_title.dart';
+import 'package:fluffychat/pangea/utils/overlay.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:matrix/matrix.dart';
+
+enum _EventContextAction { info, report }
```

Comment:
This enum is already defined in chat_view.dart and doesn't need to be redefined here. I think because it's a private enum it can't be imported without changing the name. If it's not mentioned anywhere else outside of this file, maybe comment the definition in chat_view.dart out and leave a little comment explaining why.